{"name": "test_buurten_failure", "status": "passed", "description": "Failure API request for the buurten", "steps": [{"name": "API Call `https://acc.api.data.amsterdam.nl/v1/gebieden/buurten/0363098000001 to validate whether the no record found with `failure code 404`", "status": "passed", "start": 1646901698312, "stop": 1646901698448}], "start": 1646901698312, "stop": 1646901698449, "uuid": "cb37d55f-48fc-4e7b-a2f7-d57e92df02bd", "historyId": "6d2d2afe8521e49849a7d1201a789880", "testCaseId": "76f46efd1f9a250d2bb8382d9d249708", "fullName": "e2etest.test_datasetgebiedenbuurten#test_buurten_failure", "labels": [{"name": "parentSuite", "value": "e2etest"}, {"name": "suite", "value": "test_datasetgebiedenbuurten"}, {"name": "host", "value": "Jayashrees-MacBook-Pro.local"}, {"name": "thread", "value": "63404-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "e2etest.test_datasetgebiedenbuurten"}]}